src/utils/distributions.cpp

```diff
@@ -18,3 +19,129 @@ double bayesmix::multi_normal_prec_lpdf(const Eigen::VectorXd &datum,
   double exp = (prec_chol * (datum - mean)).squaredNorm();
   return 0.5 * (base - exp);
 }
+
+double bayesmix::gaussian_mixture_dist(
+    Eigen::VectorXd means1, Eigen::VectorXd vars1, Eigen::VectorXd weights1,
+    Eigen::VectorXd means2, Eigen::VectorXd vars2, Eigen::VectorXd weights2) {
```

Comment:
If I understand correctly, this is the distance between two normal mixtures. We should clarify this better, maybe by changing dist into distance or even with a single line of documentation comment

Reply:
I added some docs